core/domain/services/graphql/types/Ticket.php

```diff
@@ -301,6 +301,12 @@ public function getFields()
                 null,
                 esc_html__('Ticket Creator ID', 'event_espresso')
             ),
+            new GraphQLField(
+                'visibility',
+                'Int',
+                'visibility',
+                esc_html__('Where the ticket can be viewed throughout the UI', 'event_espresso')
+            ),
```

Comment:
Since the client needs to be aware of server-side integer values, it's better to make it `Enum`, instead of `Int`

Reply:
Here is the difference between the two.
![image](https://user-images.githubusercontent.com/18226415/117428093-5c3b6c80-af43-11eb-87e7-107e10a4d27f.png)


With enum, mutations are pretty nice
![image](https://user-images.githubusercontent.com/18226415/117428144-69f0f200-af43-11eb-95c8-093dc6656c54.png)